Dockerfile.nginx

```diff
@@ -0,0 +1,15 @@
+FROM nginx:1.13.3
+
+MAINTAINER Conjur Inc
+
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && \
+    apt-get install -y iputils-ping procps openssl
+
+WORKDIR /etc/nginx/
+
+RUN openssl req -x509 -nodes -days 365 -newkey rsa:2048 \
+    -subj "/C=IL/ST=Israel/L=TLV/O=Onyx/CN=conjur-proxy-nginx" \
+    -keyout cert.key -out cert.crt
+
+ADD /test-proxy/default.conf /etc/nginx/conf.d/default.conf
```

Comment:
- Can you add a newline at the end of each text file? Kumbi pointed that out and it's explained here https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
yes please

Reply:
Also, I don't understand why we need this nginx image. Can you clarify?

Reply:
I will change the ADD command to copy,
About the nginx this is the easy way I found how to automatically run ssl tests against the conjur server, the nginx configured to work https and forwarded the requests to conjur by http